pkgs/development/python-modules/mariadb/default.nix

```
@@ -0,0 +1,26 @@
+{ buildPythonPackage, fetchPypi, libmysqlclient, lib, pythonOlder }:
+buildPythonPackage rec {
+  pname = "mariadb";
+  version = "1.0.9";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Aqmz0KB26aDQ6hxItF7Qm2R14rak6Mge2fHoLK87/Ck=";
+    extension = "zip";
+  };
+
+  nativeBuildInputs = [
+    libmysqlclient
+  ];
+
+  pythonImportsCheck = [ "mariadb" ];
```

```suggestion
  # Requires a running MariaDB instance
  doCheck = false;

  pythonImportsCheck = [ "mariadb" ];
```

No tests are run. Thus, we can skip `checkPhase`.

```bash
running build_ext
copying build/lib.linux-x86_64-3.9/mariadb/_mariadb.cpython-39-x86_64-linux-gnu.so -> mariadb

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```